spec/commands/order/reminder_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'commands/order/reminder'
 require 'fake_channel_info_provider'
+require 'spec_helper'
```

Comment:
How come this file needs to be added everywhere now? Doesn't your '.rspec` file contain `--require spec_helper` anymore?